### PR TITLE
Re-enabling pre-commit eslint

### PR DESCRIPTION
**Pull request by Samuel-Therrien-Beslogic**

### CODE REVIEW

**Comment by Samuel-Therrien-Beslogic** [comment]
I wonder if pre-commit.ci infinite queue is caused by https://www.githubstatus.com/incidents/88003ngctf1z ?

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -21,25 +22,53 @@ repos:
...
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: v8.56.0
+    hooks:
+      - id: eslint
+        args: [--fix, --quiet]
+        files: \.[jt]sx?$ # *.js, *.jsx, *.ts and *.tsx
+        types: [file]
+        additional_dependencies: [
+            # Keep these in sync with canopeum_frontend/package.json
+            "eslint-config-beslogic@2.4.4",
```
Comment (Samuel-Therrien-Beslogic) [inline .pre-commit-config.yaml:36]:
```suggestion
            "eslint-config-beslogic@2.4.5",
```

**Comment by Samuel-Therrien-Beslogic** [comment]
pre-commit.ci run

**Comment by Samuel-Therrien-Beslogic** [comment]
As per pre-commit maintainer's request, let's avoid pushing or updating this branch at all. So we don't make it a moving target.
https://github.com/pre-commit-ci/issues/issues/223#issuecomment-2236427229

**Comment by asottile** [comment]
pre-commit.ci run

**Comment by Samuel-Therrien-Beslogic** [comment]
Closing in favor of https://github.com/BesLogic/releaf-canopeum/pull/202 as using the mirror repo causes a bigger bundle which *just* goes over the limit

**Pull request closed without merging**